### ISSUE TITLE
curve result

### ISSUE THREAD

**Issue by miao02830** [dongkwonjin/Eigenlanes#6]
This paper mentioned that the model proposed in this paper is more effective in curve scene testing. But from the results, on the culane dataset, the detection accuracy of curve category is not high, and it is lower than other models. How to explain this? Thank you very much.

**Comment by dongkwonjin** [comment]
Hi, Thank you for your attention!

As you pointed out, on the 'curve' category, the proposed method yields poor results to the other methods.
But, this is because the CULane training data are not enough to generate curved lane candidates. Please note that the proportion of curved lanes is only 1.2% on CULane.

This issue is also discussed in the paper (Section 4.3).
